CHANGELOG.md

```diff
@@ -16,11 +16,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Adds Storybook configs (#463)
 - Adds vtex search tracking script. With this we will populate TopSearches and Autocomplete indices (#389)
 - Add `RegionalizationBar`, `RegionalizationButton` components and integrates it on Mobile and Desktop devices (#424).
+- Integrates and styles ImageGallery component at the PDP (#409)
 
 ### Changed
 
 - Applies new local tokens to `BannerText` (#470)
 - Update the Incentives component to handle CMS data (#474)
+- Refactors ImageGallery component to not use the useSlider component (#409)
```

Comment:
Can you move these new entries to the beginning of their sections? [The latest version comes first.](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.).